controllers/mentor.go

```diff
@@ -17,6 +18,32 @@ type RegisterMentorReqFields struct {
 	Email    string `json:"email"`
 }
 
+type ProjectInfo struct {
+	Name     string `json:"name"`
+	RepoLink string `json:"repo_link"`
```

Comment:
Missed this earlier, but it would be really helpful if the project status is also sent in the response to be displayed on the dashboard.